src/librawspeed/decompressors/LJpegPlain.cpp

```diff
@@ -259,9 +259,7 @@ void LJpegPlain::decodeN_X_Y() {
 
     if (X_S_F == 1) // will be optimized out
       for (uint32 i = 0; i < skipX; i++)
-        unroll_loop<N_COMP>([&](int i) {
-          ht[i]->decodeNext(bitStream);
-        });
+        unroll_loop<N_COMP>([&](int j) { ht[j]->decodeNext(bitStream); });
```

Comment:
Please don't 'inline' the loop body. It decreases readability because you have to parse the lambda syntax in your head to see what is actually going on inside, while in the old code the indentation showed you that.